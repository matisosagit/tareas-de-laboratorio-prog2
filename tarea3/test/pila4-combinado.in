crearPila
apilarPila 22
cimaPila
apilarPila 21
imprimirPila
apilarPila 20
cantidadPila
apilarPila 19
cantidadPila
imprimirPila
desapilarPila
imprimirPila
cimaPila
cantidadPila
liberarPila
Fin