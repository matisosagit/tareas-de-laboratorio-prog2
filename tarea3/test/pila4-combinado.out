1> Se ejecutó 'crearTPila' exitosamente
2> Se ejecutó 'apilarTPila' exitosamente con entero 22
3> Elemento obtenido al ejecutar cima: 22.
4> Se ejecutó 'apilarTPila' exitosamente con entero 21
5> Pila:
Pila: 22 21
6> Se ejecutó 'apilarTPila' exitosamente con entero 20
7> Existen 3 elementos en la pila.
8> Se ejecutó 'apilarTPila' exitosamente con entero 19
9> Existen 4 elementos en la pila.
10> Pila:
Pila: 22 21 20 19
11> Se ejecutó 'desapilarTPila' exitosamente
12> Pila:
Pila: 22 21 20
13> Elemento obtenido al ejecutar cima: 20.
14> Existen 3 elementos en la pila.
15> Se ejecutó 'liberarTPila' exitosamente.
16> Fin.
